src/App.js

```diff
@@ -1,15 +1,49 @@
+import React from 'react';
+import { useState } from 'react';
 import './App.css';
 
 const App = () => {
 
+  const [count, setCount] = useState(0);
+  const [counterText, setCounterText] = useState('');
+
+  const addCounter = () => {
+    setCount(prevCount =>  prevCount  + 1)
+  }
+
+  const subtractCounter = () => {
+    setCount((prevCount) => prevCount - 1)
+  }
+
+  const SetCountText = (event) => {
```

Comment:
use proper naming convention functions start with lower case

Reply:
also the function is supposed to be handleCounterTextChange as that is what is being done